### PR TITLE
Addresses HT-1455 display institution name.

### PR DESCRIPTION
**Pull request by moseshll**
Part of the ticket relates to grouping by institution for batch operations. Since sorting by columns is not implemented yet, I'm taking that as not part of the MVP for the current effort.

### CODE REVIEW

File: app/models/ht_institution.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# We're not really interested in editing or viewing this table,
+# but we want to allow HTUser access to a human-readable version of
+# the institution.
+class HTInstitution < ApplicationRecord
+  self.primary_key = 'sdrinst'
```
Comment (aelkiss) [inline app/models/ht_institution.rb]: `sdrinst` is deprecated; use `inst_id` instead.

File: db/schema.rb [hunk 2]
```diff
@@ -31,4 +33,24 @@
     t.string :identity_provider
   end
 
+  create_table :ht_institutions, id: false do |t|
+    t.string :sdrinst, primary_key: true
+    t.string :inst_id
```
Comment (aelkiss) [inline db/schema.rb]: As above, `sdrinst` is deprecated

**Comment by aelkiss** [comment]
I'll also deploy and check that this works as expected in dev.

File: app/models/ht_institution.rb (hunk 1 above)
Comment (moseshll) [reply app/models/ht_institution.rb]: Should be addressed in second commit. Tests pass locally.

File: db/schema.rb (hunk 2 above)
Comment (moseshll) [reply db/schema.rb]: Ditto above.

**Comment by moseshll** [comment]
I'm not sure this will work if deployed to dev with the Shib branch unmerged.

**Comment by aelkiss** [comment]
Works so long as you use the WAYFless link (that I had posted a while back in Slack) to force login. The column widths could be better but this will work for now. I'll go ahead and merge it.

**Pull request merged**